.changeset/wild-days-jog.md

```diff
@@ -0,0 +1,11 @@
+---
+"@cloudflare/vite-plugin": patch
+---
+
+fix node.js compat module resolution
+
+The polyfills that get compiled into the Worker were not being found
+because they were dependencies of the plugin, not the user's project.
```

Comment:
This has always been the case so I don't think it explains why the issue occurred and under what conditions (package manager etc.).

Reply:
Updated - hopefully gives enough context.